superset/connectors/sqla/models.py

```diff
@@ -145,6 +147,26 @@ class TableColumn(Model, BaseColumn):
     update_from_object_fields = [s for s in export_fields if s not in ("table_id",)]
     export_parent = "table"
 
+    def get_expression(self):
+        if config["DEFAULT_DB_DTTM_ENABLED"] and not self.expression:
+            # TODO(bkyryliuk): consider serializing defaults to the DB for better visibility.
```

Comment:
Is this TODO going to be handled in the near future? If not, I'd recommend removing it.

Reply:
yes in near future if there are no objections